internal/utils/utils.go

```diff
@@ -212,8 +213,7 @@ func CloneRepo(stagePath string, repo string) string {
 // DieIfError kills the program if err is not nil.
 func DieIfError(err error, message string) {
 	if err != nil {
-		fmt.Println(message)
-		os.Exit(1)
+		log.Fatalf("%s.\nFailed with error: %v", message, err)
```

Comment:
I think log should be used for printing errors. It exits the program afterwards and prints to stderr instead of stdout which is really where those errors should be ending up. I feel like I've seen this a fair bit in Go projects where the program exit is handled in the caller with a `log.Fatal`. I checked around though and am seeing many people like to use `os.Exit` if they're not using `log` for anything else. 

Tbh I'm good with either 🤷